docs/whats-new/index.yml

```diff
@@ -0,0 +1,35 @@
+### YamlMime:Landing
+
+title: .NET Documentation What's New?
+summary: "Welcome to What's new in .NET docs. Quickly find the latest changes in .NET docs."
+
+metadata:
+  title: .NET Documentation What's New?
+  description: "Welcome to What's new in .NET docs."
+  ms.topic: landing-page
+  ms.date: 12/12/2019
+
+landingContent:
+  - title: "What's new"
+    linkLists:
+      - linkListType: whats-new
+        links:
+          - text: November 2019
+            url: 2019-11.md
+
+  - title: "Open source contributions"
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: ".NET Docs repositories overview."
+            url: https://github.com/dotnet/docs/blob/master/README.md
+          - text: "Understand our project structure and label usage for issues and PRs."
+            url: https://github.com/dotnet/docs/blob/master/styleguide/labels-projects.md
+      - linkListType: concept
+        links:
+          - text: ".NET docs Contributor guide"
+            url: https://github.com/dotnet/docs/blob/master/CONTRIBUTING.md
+          - text: ".NET API reference docs Contributor guide"
+            url: https://github.com/dotnet/dotnet-api-docs/blob/master/CONTRIBUTING.md
+          - text: ".NET samples Contributor guide"
+            url: https://github.com/dotnet/samples/blob/master/CONTRIBUTING.md
```

Comment:
Should we have another card with links to what's new in docs for other docsets, e.g. ASP.NET, Xamarin?

Reply:
I like that idea. What do you think @davidbritch?

Reply:
Especially because with two cards the layout looks weird.

Reply:
> I like that idea. What do you think @davidbritch?

Sounds good. Also happy to link to your page from our what's new docs.

Reply:
Latest commit has this added.